.github/workflows/ansible-tests-pr.yml

```diff
@@ -31,7 +31,7 @@ jobs:
       run:
         working-directory: .ansible/collections/ansible_collections/equinix/cloud
     steps:
-      - name: checkout Ansible collection
+      - repository: checkout Ansible collection
```

Comment:
This doesn't appear to be a valid repository value:

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-using-an-action-inside-a-different-private-repository-than-the-workflow

Based on the value, I would think you may have meant to replace `name` with [`run_name`](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#run-name), but not clear that that would fix anything

Reply:
@displague thanks for pointing it out. I wanted to change the `name` param or the checkout action. `name` there is invalid param, it should be `repository`.